interactive_demo.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+
+import math
+import os
+
+from drake_ros.systems import MoveableJoints
+from drake_ros.systems import MoveablePoint
+from drake_ros.systems import SystemClock
+from drake_ros.systems import TFPublisher
+
+from interactive_markers import InteractiveMarkerServer
+
+import numpy
+
+from pydrake.common.eigen_geometry import Quaternion
+from pydrake.common.value import AbstractValue
+from pydrake.examples.manipulation_station import ManipulationStation
+from pydrake.geometry import ConnectDrakeVisualizer
+from pydrake.manipulation.planner import DifferentialInverseKinematicsParameters
+# from pydrake.manipulation.planner import DifferentialInverseKinematicsIntegrator
```

Comment:
Can you run `flake8 --select=F` on these files?

Reply:
Removed unused imports in 4641a7b